DCCpp_Uno/src/Config.h

```diff
@@ -11,9 +11,12 @@ Part of DCC++ BASE STATION for the Arduino
 //
 // DEFINE MOTOR_SHIELD_TYPE ACCORDING TO THE FOLLOWING TABLE:
 //
-//  0 = ARDUINO MOTOR SHIELD          (MAX 18V/2A PER CHANNEL)
-//  1 = POLOLU MC33926 MOTOR SHIELD   (MAX 28V/3A PER CHANNEL)
-//  2 = BTS7960B                      (MAX 27V/43A PER CHANNEL)
+//  0 = ARDUINO MOTOR SHIELD            (MAX 18V/2A  PER CHANNEL)  Arduino Motor shield Rev3 based on the L298
+//  1 = POLOLU MC33926 MOTOR SHIELD     (MAX 28V/2.5 PER CHANNEL)  Pololu MC33926 Motor Driver (shield or carrier)
+//  2 = BTS7960B_5A                     (MAX 27V/5A  PER CHANNEL)  Infineon Technologies BTS 7960 Motor Driver Module. Max Output 5A (43A actual max)
+//  3 = BTS7906B_10A                    (MAX 27V/10A PER CHANNEL)  Infineon Technologies BTS 7960 Motor Driver Module. Max Output 10A (43A actual max
+//  4 = LMD18200 MOTOR DRIVER MODULE
+//      & MAX 471 CURRENT SENSE MODULE  (MAX 28V/3A  PER CHANNEL)
```

Comment:
Good catch. I tried it with the 471 because I saw it on the locoduino site . So I can change that, but wondering then if we have the 471 as an option on its own since it can connect to anything, or if we modularize this. IOW, since the locoduino version is 2 LMD18200s and 2 MAX471s, people see a quick start page with connections and then just select that option from the list. Much like we are doing with the 5 and 10 Amp versions of the BTS7906

Reply:
Locoduino may be opting for the MAX471 for other reasons. It could be listed on it's own but it isn't an h-bridge but instead a current sense device. I don't think it makes sense to list it here really.